### DISCUSSION TITLE
Is there a document template for the complete record of one animal?

### DISCUSSION THREAD

**Discussion by CUKim104** [sheltermanager/asm3#1477] [Q&A / Help] [answered]
I need to have a document with the entire record of a single animal. I thought I saw a way to do this, but now I can't find it.

**Comment by bobintetley** [comment, marked as answer]
You can use "Print Animal Record" from Reports->Auditing. If you don't have that report available, you can install it under Settings->Reports->Browse sheltermanager.com

**Reply by CUKim104** [reply]
Thank you!
